fmriprep/workflows/bold/t2s.py

```diff
@@ -89,7 +90,7 @@ def init_bold_t2s_wf(echo_times,
     workflow.connect([
         (inputnode, t2smap_node, [('bold_file', 'in_files')]),
         (t2smap_node, outputnode, [('optimal_comb', 'bold')]),
-        (t2smap_node, skullstrip_t2smap_wf, [('t2star_adaptive_map', 'inputnode.in_file')]),
+        (t2smap_node, skullstrip_t2smap_wf, [('t2star_map', 'inputnode.in_file')]),
```

Comment:
shouldn't this be:
```suggestion
        (t2smap_node, skullstrip_t2smap_wf, [('t2s_vol', 'inputnode.in_file')]),
```

as per https://github.com/poldracklab/fmriprep/blob/a783c38c72cceadaf4ef0ddb6abfee947b184a2f/fmriprep/interfaces/multiecho.py#L129?

Reply:
This got updated in #1296 ! Now it's: https://github.com/poldracklab/fmriprep/blob/1c25175980522ab037366b3055797408b4709fe9/fmriprep/interfaces/multiecho.py#L45

We also return quite a few more outputs, now !